src/udev/udev-dump.c

```diff
@@ -28,7 +29,202 @@ void event_cache_written_sysctl(UdevEvent *event, const char *attr, const char *
+                }
+
+                r = sd_json_variant_set_field_strv(&v, "tags", strv_sort(tags));
+                if (r < 0)
+                        return r;
+        }
+
+        if (sd_device_get_devnum(dev, NULL) >= 0) {
+
+                if (sd_device_get_devlink_first(dev)) {
+                        _cleanup_strv_free_ char **links = NULL;
+                        int prio = 0;
+                        (void) device_get_devlink_priority(dev, &prio);
+
+                        FOREACH_DEVICE_DEVLINK(dev, devlink) {
+                                r = strv_extend(&links, devlink);
+                                if (r < 0)
+                                        return r;
+                        }
+
+                        r = sd_json_variant_set_field_strv(&v, "symlinks", strv_sort(links));
+                        if (r < 0)
+                                return r;
+                }
+
+                r = sd_json_variant_merge_objectbo(
+                                &v,
+                                SD_JSON_BUILD_PAIR_BOOLEAN("inotifyWatch", event->inotify_watch));
+                if (r < 0)
+                        return r;
+
+                uid_t uid = event->uid;
+                if (!uid_is_valid(uid))
+                        (void) device_get_devnode_uid(dev, &uid);
+                if (uid_is_valid(uid)) {
+                        _cleanup_free_ char *user = uid_to_name(uid);
+                        if (!user)
+                                return -ENOMEM;
+
+                        _cleanup_(sd_json_variant_unrefp) sd_json_variant *w = NULL;
+                        r = sd_json_buildo(
+                                        &w,
+                                        SD_JSON_BUILD_PAIR_UNSIGNED("uid", uid),
+                                        JSON_BUILD_PAIR_STRING_NON_EMPTY("name", user));
+                        if (r < 0)
+                                return r;
+
+                        r = json_variant_set_field_non_null(&v, "owner", w);
+                        if (r < 0)
+                                return r;
+                }
+
+                gid_t gid = event->gid;
+                if (!gid_is_valid(uid))
+                        (void) device_get_devnode_gid(dev, &gid);
+                if (gid_is_valid(gid)) {
+                        _cleanup_free_ char *group = gid_to_name(gid);
+                        if (!group)
+                                return -ENOMEM;
+
+                        _cleanup_(sd_json_variant_unrefp) sd_json_variant *w = NULL;
+                        r = sd_json_buildo(
+                                        &w,
+                                        SD_JSON_BUILD_PAIR_UNSIGNED("gid", gid),
+                                        JSON_BUILD_PAIR_STRING_NON_EMPTY("name", group));
+                        if (r < 0)
+                                return r;
+
+                        r = json_variant_set_field_non_null(&v, "group", w);
```

Comment:
hmm, the user you place in a "owner" field, but the group in a "group" field? that sounds weirdly uneven?

maybe "ownerUser", "ownerGroup"? (or is there prior art otherwise?)